tests/test_getters.py

```diff
@@ -69,4 +69,28 @@ def test_single_bibliography_existing(client, session):
     assert res.status == 200
     # And the response is non empty
     assert res.json().get('eid') == '123'
-    assert res.json().get('description') == 'lorem'
\ No newline at end of file
+    assert res.json().get('description') == 'lorem'
+
+
+def test_document_endpoint_eid_not_existing(client, session):
+    # Given some records matching records in the database
+    bib = Bibliography(eid='123', description='lorem')
+    session.add(bib)
+    session.flush()
+    doc = Document(
+        eid='345',
+        bibliography_eid='123',
+        title='lorem',
+        keywords='{}',
+        description='lorem',
+        language='english',
+        hash='alksdjksjdf'
+    )
+    session.add(doc)
+    session.commit()
```

Comment:
This bit up there is unnecessary, since this assertion can be made with an empty database.